Stripe/STPAPIClient.m

```diff
@@ -312,15 +314,33 @@ - (void)retrieveSourceWithId:(NSString *)identifier clientSecret:(NSString *)sec
     NSCAssert(identifier != nil, @"'identifier' is required to create a source");
     NSCAssert(secret != nil, @"'secret' is required to create a source");
     NSCAssert(completion != nil, @"'completion' is required to use the source that is created");
+    [self retrieveSourceWithId:identifier clientSecret:secret responseCompletion:^(STPSource * object, __unused NSHTTPURLResponse *response, NSError *error) {
+        completion(object, error);
+    }];
+}
+
+- (NSURLSessionDataTask *)retrieveSourceWithId:(NSString *)identifier clientSecret:(NSString *)secret responseCompletion:(STPAPIResponseBlock)completion {
     NSString *endpoint = [NSString stringWithFormat:@"%@/%@", sourcesEndpoint, identifier];
     NSDictionary *parameters = @{@"client_secret": secret};
-    [STPAPIRequest<STPSource *> getWithAPIClient:self
-                                        endpoint:endpoint
-                                      parameters:parameters
-                                      serializer:[STPSource new]
-                                      completion:^(STPSource *object, __unused NSHTTPURLResponse *response, NSError *error) {
-                                          completion(object, error);
-                                      }];
+    return [STPAPIRequest<STPSource *> getWithAPIClient:self
+                                               endpoint:endpoint
+                                             parameters:parameters
+                                             serializer:[STPSource new]
+                                             completion:completion];
+}
+
+- (void)startPollingSourceWithId:(NSString *)identifier clientSecret:(NSString *)secret completion:(STPSourceCompletionBlock)completion {
+    [self stopPollingSourceWithId:identifier];
```

Comment:
Header doc should make it clear that you can only have 1 poll per source at a time (or change this to be able to fire multiple completion blocks from one polling, although we might have some threading issues if we make it more complicated like this).